crates/host/src/wasmbus/host_config.rs

```diff
@@ -33,8 +33,8 @@ pub struct Host {
     pub rpc_key: Option<Arc<KeyPair>>,
     /// Whether to require TLS for RPC connection
     pub rpc_tls: bool,
-    /// The lattice the host belongs to
-    pub lattice: Arc<str>,
+    /// The lattices the host belongs to
+    pub lattices: Arc<[Box<str>]>,
```

Comment:
I know slices/vecs are a lot easier to work with, but since this is the "source of truth" for what lattices a host belongs to, it might be a good idea to make this a HashSet<> to avoid duplicates since this is a public field & can be constructed directly.

Changing here would let us use a `Vec` wherever we wanted in the code (or a `Arc<[Box<str>]>` if that's the type we keep around in memory) but know the values are unique and avoid setting up duplicate subscriptions